Project2015To2017/Transforms/PropertySimplificationTransformation.cs

```diff
@@ -212,6 +215,14 @@ bool ValidateConditionedDefaultValue(string @default)
 				{
 					return (valueLower == @default) && (!hasGlobalOverride || globalOverrideLower == @default);
 				}
+				
+				bool ValidateDefaultVersion()
+				{
+					return ValidateDefaultValue("1.0.0.0")
```

Comment:
What if user specifies 1.0.0.0.0? I mean, who would be dumb enough, but should we handle that as well?
Like continuously `TrimEnd` `.0` and see if only `1` is left?

Reply:
I went for covering valid version numbers. 1.0.0.0.0 wouldn't have worked in the old world so I don't think we need to bother covering it whereas 1.0.0 would have worked and I think 1.0 and 1 would have too.

Reply:
Ok, seems good. If user does stupid things there is no need to nurture them.